js/jquery.keyboard.js

```diff
@@ -753,6 +753,11 @@ http://www.opensource.org/licenses/mit-license.php
 					k1 = k >= keyCodes.A && k <= keyCodes.Z,
 					k2 = k >= keyCodes.a && k <= keyCodes.z,
 					str = base.last.key = String.fromCharCode(k);
+				// check, that keypress wasn't rise by functional key
+				// space is first typing symbol in UTF8 table
+				if (k < keyCodes.space && e.keyCode !== 0){ //see #549
```

Comment:
I force upload to add validation of e.keyCode for hard control of that we on keypress event with functional key.